src/runes.c

```diff
@@ -8,7 +8,8 @@ void RegenLostRot();
 void RuneRespawn();
 void RuneTouch();
 void RuneResetOwner();
-gedict_t* SelectRuneSpawnPoint();
+gedict_t* SelectSpawnPoint();
```

Comment:
I think we don't need this.
The prototype of `SelectSpawnPoint()` is already in g_local.h, which is included in this file.
But in any case, the proper function definition is `gedict_t* SelectSpawnPoint(char *spawnname);`